build/images/katas/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# We use the iqsharp-base image, as that includes
+# the .NET Core SDK, IQ#, and Jupyter Notebook already
+# installed for us.
+FROM mcr.microsoft.com/quantum/iqsharp-base:0.11.2004.2715-beta
+
+# Add metadata indicating that this image is used for the katas.
+ENV IQSHARP_HOSTING_ENV=KATAS_DOCKERFILE
+
+# Make sure the contents of our repo are in ${HOME}
+# Required for mybinder.org
+COPY . ${HOME}
+
+USER root
+RUN chown -R ${USER} ${HOME} && \
+    chmod +x ${HOME}/scripts/*.sh
+
+# Install Python dependencies for the Python visualization and tutorial notebooks
+RUN pip install -I \
+        matplotlib \
+        pytest
+
+USER ${USER}
+
+# Pre-exec notebooks to improve first-use start time
+RUN cd $HOME && \
```

Comment:
Consider using `pushd` and `popd` instead of `cd`? Also consider `${HOME}` for consistency with the rest of the file?